app/lib/actions/foreman_ansible/play_host_roles.rb

```diff
@@ -0,0 +1,46 @@
+module Actions
+  module ForemanAnsible
+    class PlayHostRoles < Actions::EntryAction
+      include ::Actions::Helpers::WithContinuousOutput
+      include ::Actions::Helpers::WithDelegatedAction
+
+      def plan(host, proxy_selector = ::ForemanAnsible::ProxySelector.new)
```

Comment:
I've realized I've forgot to push thte proxy selector class to the git :(, it's there now